marge/batch_job.py

```diff
@@ -354,4 +346,4 @@ def execute(self):
                 log.info('batch_mr.accept result: %s', ret)
             except gitlab.ApiError as err:
                 log.exception('Gitlab API Error:')
-                raise CannotMerge('Gitlab API Error: %s' % err) from err
+                raise CannotMerge(f'Gitlab API Error: {err}') from err
```

Comment:
Silly question, but what does `from err` mean here? That it will reuse the stack trace from the `err` exception?

Reply:
Looks like it's about exception chaining: https://docs.python.org/3/tutorial/errors.html#exception-chaining